src/templates/blog.js

```diff
@@ -19,53 +21,88 @@ export default function NewsIndex({ data, pageContext }) {
+                content: "https://ebpf.io/blog",
+              },
+              { name: "title", property: "og:title", content: pageMetaTitle },
+              {
+                name: "description",
+                property: "og:description",
+                content: pageMetaDescription,
+              },
+              { name: "description", content: pageMetaDescription },
+              { name: "image", property: "og:image", content: pageMetaImageUrl },
+              { name: "twitter:card", content: "summary_large_image" },
+              { name: "twitter:url", content: "https://ebpf.io/blog" },
+              { name: "twitter:title", content: pageMetaTitle },
+              { name: "twitter:description", content: pageMetaDescription },
+              { name: "twitter:image", content: pageMetaImageUrl },
+            ]}
+          >
+            <link
+              rel='alternate'
+              type='application/rss+xml'
+              title='RSS Feed'
+              href='/blog/rss.xml'
+            />
+          </Helmet>
 
-        {group.map(({ node: post }) => (
-          <PostCard key={post.id} post={post} />
-        ))}
-        <div className='prev-next-links blog-post'>
-          <div className='prev-link'>
-            {!first && (
-              <NavLink url={`/blog/${prevUrl}`} text='« Newer Posts' />
-            )}
+          <div className="categories-wrapper">
+            <h2 className="categories-heading">Categories</h2>
+            <ul className="categories-item-list">
+              <li className='categories-item'>
+                <a className="categories-link" href='/blog/'>
+                  All posts
+                </a>
+              </li>
+                {uniqueCategories.map(
+                  (category, _) =>
+                    category !== "_" && (
+                      <li className='categories-item' key={category}>
+                        <a className="categories-link" href={`/blog/categories/${slugify(category, {lower: true})}`}>
+                          {category}
+                        </a>
+                      </li>
+                    )
+                )}
+            </ul>
           </div>
-          <div className='next-link'>
-            {!last && <NavLink url={`/blog/${nextUrl}`} text='Older Posts »' />}
+
+          <div className="posts-wrapper">
+          {featuredPost && 
+          (<>
+            <h2 className="posts-heading">Featured</h2>
+            <FeaturedPostCard post={featuredPost}/>
+          </>)}
+
+          <h2 className="posts-heading">Latest</h2>
+          {group.filter((post) => post.node.frontmatter.isFeatured !== true).map(({ node: post }) => (
+            <PostCard key={post.id} post={post} />
+          ))}
+            <div className='prev-next-links blog-post'>
+              <div className='prev-link'>
+                {!first && (
+                  <NavLink url={`/blog/${prevUrl}`} text='« Newer Posts' />
```

Comment:
As you return nothing, this component doesn't work properly. Should be changed to the code below. 
```
const NavLink = (props) => <Link to={props.url}>{props.text}</Link>;
```